code/identity/build.gradle.kts

```diff
@@ -18,7 +18,9 @@ aepLibrary {
         mavenRepoName = identityMavenRepoName
         mavenRepoDescription = identityMavenRepoDescription
         gitRepoName = "aepsdk-core-android"
-        addCoreDependency(coreExtensionVersion)
+
+        // Stop using snapshot version after release
+        addCoreDependency("$coreExtensionVersion-SNAPSHPOT")
```

Comment:
if we are planning to hardcode "-SNAPSHPOT".. 
Do you think if it is better to suffix them in the mavenCoreVersion property of gradle.properties file

```
mavenCoreVersion=3.0.0-beta.1-SNAPSHOT
```

Reply:
Core project use the same gradle.`coreExtensionVersion` instead of the `mavenCoreVersion` property. This is also used for `core` build.gradle. Adding another `mavenCoreVersion` property is going to be confusing. So,I prefer this approach for this project.
Other extensions can change the `mavenCoreVersion` as needed.

Reply:
I agree with @prudrabhat , in most cases, we won't need to use another `mavenCoreVersion` property. Specifying the dependency by adding  `addCoreDependency("$coreExtensionVersion-SNAPSHPOT")` is very straightforward, and we can easily comment it out for future releases.